controllers/operands/ssp.go

```diff
@@ -297,6 +314,22 @@ func getCustomDicts(list []hcov1beta1.DataImportCronTemplateStatus, crDicts map[
 	return list
 }
 
+func filterEnabledDicts(list []hcov1beta1.DataImportCronTemplateStatus) []hcov1beta1.DataImportCronTemplateStatus {
+	if list == nil {
+		return nil
+	}
+
+	result := make([]hcov1beta1.DataImportCronTemplateStatus, 0, len(list))
+	for _, dictStatus := range list {
+		if !isDataImportCronTemplateEnabled(dictStatus.DataImportCronTemplate) {
+			continue
+		}
+		result = append(result, dictStatus)
+	}
+
+	return result
+}
+
```

Comment:
It will be so much easy and nice code in golang 1.23... we'll get there :)

Reply:
Yes, iterators will be nice.